schemas/6s4t_run_stop.fbs

```diff
@@ -10,10 +10,11 @@
 file_identifier "6s4t";
 
 table RunStop {              //  *Mantid*    // *File Writer* // *Description*
-    stop_time : uint64;      //  Required    //  Required     // milliseconds since Unix epoch (1 Jan 1970)
+    stop_time : uint64;      //  Required    //  Required     // milliseconds since Unix epoch (1 Jan 1970). When set to 0, will trigger a "stop NOW" code path in the file writer.
     run_name : string;       //  Required    //  Unused       // Name for the run, must match corresponding field in RunStart
     job_id : string;         //  Unused      //  Required     // A unique identifier for the file writing job
     service_id : string;     //  Unused      //  Optional     // The identifier for the instance of the file-writer that should handle this command
+    command_id : string;     //  Unused      //  Required     // Required for uniquely identifying a stop command. Enforcement of the uniqueness will be limited but done by the filewriter.
```

Comment:
I'm not sure what "Enforcement of the uniqueness will be limited but done by the filewriter." means. Perhaps some practical advice in the comment would be more useful? For example "generating a `uuid4` with Python is sufficient".

Reply:
It was my intention to have this be documented in the file-writer documentation. However, as it is not written yet (neither is the implementation), I can not link to it. Alternatively, I can write some documentation here but I worry that it will be of limited usefulness as it is (even more) likely that it will be outdated due to it being separated from the actual implementation.

Reply:
Another alternative is of course to remove any mentioning of uniqueness enforcement. It is something that only the file-writer cares about after all.

Reply:
I think it is too cryptic in its current form. Best to remove it or be more concrete about what it means. I'm struggling to imagine what the file writer could usefully check...

Reply:
```suggestion
    command_id : string;     //  Unused      //  Required     // Required for uniquely identifying a stop command.
```